__init__.py

```
@@ -49,18 +50,31 @@ def handle_deaths_intent(self, message):
 
+    @staticmethod
+    def pronounce_year(dialog: str, lang: str) -> str:
+        try:
+            bc = False
+            year, utt = dialog.split(" - ")  # this is how .dialog files are formatted
+            if len(year.split()) == 2:
+                year, bc = year.split()
+            if year.isdigit():
+                dt = datetime.datetime(year=int(year), day=1, month=1)
+                return f"{nice_year(dt, lang=lang, bc=bool(bc))} - {utt}"
+        except:
```

Ruff

Do not use bare `except`

(E722)